publish/README.md

```diff
@@ -40,16 +43,37 @@ afterEvaluate {
       }
     }
 }
+```
+
+Next, you can apply either the `config/publish/maven-central.gradle` or `config/publish/artifactory.gradle` scripts to configure the repos where your artifacts will be published.
+
+#### Maven Central
 
+If you wish to publish snapshots, you can use the `MAVEN_CENTRAL_IS_SNAPSHOT` property if your current version is a snapshot version.
+
+```groovy
 def isSnapshot = project.version.contains('-')
 if (isSnapshot) {
-  apply from: '../config/publish/artifactory.gradle'
+  ext["MAVEN_CENTRAL_IS_SNAPSHOT"] = true
+}
+apply from: '../config/publish/maven-central.gradle'
+```
+
+#### Artifactory
+
+If you wish to publish snapshots, you can publish to your snapshot URL if your current version is a snapshot version.
+
+```groovy
+def isSnapshot = project.version.contains('-')
+if (isSnapshot) {
+  ext["ARTIFACTORY_URL"] = System.getenv("ARTIFACTORY_URL_RELEASE")
```

Comment:
I think this should be `MAVEN_CENTRAL_URL_SNAPSHOT`.

Reply:
Oh, actually this one is correct because this is the Artifactory section. The one below should be changed to `ARTIFACTORY_URL_SNAPSHOT`. Thanks, I'll update this.

Reply:
Ah sorry, now I see that both of them need to be fixed: this one should `ARTIFACTORY_URL_SNAPSHOT` and the other should be `ARTIFACTORY_URL_RELEASE`.